proteus/WaveTools.pyx

```diff
@@ -420,20 +691,48 @@ class MonochromaticWaves:
 
 
 class RandomWaves:
-    """Generate approximate random wave solutions
-    :param Tp: frequency [1/T]
-    :param Hs: significant wave height [L]
-    :param mwl: mean water level [L]
-    :param  depth: depth [L]
-    :param waveDir:wave Direction vector with three components [-]
-    :param g: Gravitational acceleration vector with three components [L/T^2]
-    :param N: number of frequency bins [-]
-    :param bandFactor: width factor for band  around fp [-]
-    :param spectName: Name of spectral function in string format. Use a random word and run the code to obtain the vaild spectra names
-    :param spectral_params: Dictionary of additional arguments for spectral function, specific to each spectral function, except from Hs and Tp e.g. {"gamma": 3.3, "TMA" = True, "depth" = 1} for Jonswap. Check spectral function arguments
-    :param phi: Array of component phases - if set to none, random phases are assigned
-"""
-
+    """
+    This class is used for generating plane random waves using linear reconstruction of components from a
+    wave spectrum
+    
+    Parameters
+    ----------    
+    param : Tp
+            Description: Peak wave period
+            Type: Float
+    param : Hs
+            Description: Significant wave height
+            Type: float
+    param : mwl
+            Description: Still water level
+            Type: float
+    param : depth
+            Description: Water depth
+            Type: float
+    param : waveDir
+            Description: Wave direction vector
+            Type: numpy array
+    param : g
+            Description: Gravitational acceleration vector 
+            Type: numpy array
+    param : N
+            Description: Number of frequency components
+            Type: integer
+    param : bandfactor
```

Comment:
The comments below in __init__ disagree with the descriptions of the parameters up here.  I figure that's probably just comments which have gotten rotten over many updates (these descriptions make a lot more sense), but I guess you should check just in case and remove unnecessary comments so they don't stick around.  I don't know how common these extra comments are - I just needed to check out this function in particular for a problem.